### PR TITLE
remove dot notation for referring to view_app

### PR DESCRIPTION
**Pull request by nickpell**
@benrudolph @emord

### CODE REVIEW

File: corehq/apps/app_manager/views/view_generic.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def view_generic(request, domain, app_id=None, module_id=None, form_id=None,
     if app and app.copy_of:
         # don't fail hard.
         return HttpResponseRedirect(reverse(
-            "corehq.apps.app_manager.views.view_app", args=[domain, app.copy_of]
+            "view_app", args=[domain, app.copy_of] # TODO - is this right?
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/view_generic.py]: E261 at least two spaces before inline comment

File: corehq/tabs/tabclasses.py [hunk 2]
```diff
@@ -596,7 +596,7 @@ def dropdown_items(self):
 
 
 class ApplicationsTab(UITab):
-    view = "corehq.apps.app_manager.views.view_app"
+    view = "default_app"
```
Comment (emord) [inline corehq/tabs/tabclasses.py]: Why are a couple of these `default_app` instead of `view_app` ?

File: corehq/apps/app_manager/views/view_generic.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/app_manager/views/view_generic.py]: @dannyroberts I wasn't sure which url should be returned here - there are many different ones that are all handled by ```corehq.apps.app_manager.views.view_app```.  Do you have an idea?

File: corehq/tabs/tabclasses.py (hunk 2 above)
Comment (nickpell) [inline corehq/tabs/tabclasses.py]: If you check out ```app_manager/urls.py``` there are many different urls that all point to the same view function ```corehq.apps.app_manager.views.view_app```.  ```default_app``` refers to ```/a/<domain>/apps/```.  This seems like an anti-pattern, but fixing that is out of scope for this set of changes.

**Review by emord — APPROVED** [review]
Looks good to me. Will let you decide on merging based on danny's response
